.eslintrc.cjs

```diff
@@ -59,6 +62,12 @@ module.exports = {
               'CallExpression[callee.name=useMemo][arguments.1.type=ArrayExpression][arguments.1.elements.length=0]',
             message:
               "`useMemo` with an empty dependency array can't provide a stable reference, use `useRef` instead."
+          },
+          {
+            // ❌ z.object(…)
+            selector:
+              'MemberExpression[object.name=z] > .property[name=object]',
+            message: 'Use z.strictObject is more safe.'
```

Comment:
z.strictObject is safer since it's disallow unknown object keys

Reply:
![image](https://user-images.githubusercontent.com/7361780/213897491-7d9b4ad2-4447-46f7-ac4e-a8f06a8fa1f7.png)